lib/cocoapods/generate/installer.rb

```diff
@@ -380,9 +378,36 @@ def print_post_install_message
             Executable.execute_command 'open', [workspace_path]
           end
         else
-          UI.info "Open #{UI.path workspace_path} to work on #{spec.name}"
+          UI.info "Open #{UI.path workspace_path} to work on it!"
         end
       end
+
+      def share_scheme(project, scheme_name)
+        scheme = Xcodeproj::XCScheme.user_data_dir(project.path) + "#{scheme_name}.xcscheme"
+        return unless File.exist?(scheme)
+        Xcodeproj::XCScheme.share_scheme(project.path, scheme_name)
+      end
+
+      def create_main_source_file(project, pod_targets, name)
```

Comment:
it is now possible for us to create a single workspace so we need to write a _single `main.m` file. with all imports in it.

We no longer use cocoapods helper method to do that because it would require us to update cocoapods to support many pod targets which would then pin us to an unreleased version of cocoapods so we take matters into our own hands here. 

it also removes some ugly cocoapods version checking too.